ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -201,10 +207,6 @@ export function PreviewConcierge(props: any) {
             }
           );
           break;
-        case INSERT_ITEM_OPERATION: {
-          enqueueSnackbar('Insert item operation not implemented.');
-          break;
-        }
```

Comment:
Why remove this?

Reply:
It says Insert item, not insert repeat group items, so I thought it was used to insert an items as insert instance did

Reply:
I added it again